README.md

```diff
@@ -127,6 +227,11 @@ When citing SCT please use this BibTeX entry:
     keywords = "Open-source ",
     }
 
+## Contributors
+
+This project exists thanks to [all the people who contribute](https://github.com/neuropoly/spinalcordtoolbox/graphs/contributors). 
+If you would like to contribute as well, please follow the [[contribution guidelines](CONTRIBUTING.md)].
```

Comment:
This is rendering with extra `[` and `]`.

Reply:
@perone thanks, now fixed